container.go

```diff
@@ -496,11 +496,11 @@ func (c *Container) fetchState(cmd string) (State, error) {
 
 func (c *Container) getSystemMountInfo() {
 	// check if /dev needs to be bind mounted from host /dev
+	c.systemMountsInfo.BindMountDev = false
+
 	for _, m := range c.mounts {
-		if m.Source == "/dev" && m.Destination == "/dev" && m.Type == "bind" {
+		if m.Destination == "/dev" && m.Source == "/dev" && m.Type == "bind" {
```

Comment:
Huh? Isn't this change is a nop (and confusing)?

Reply:
@amshinde Could you please revert that change so that we can merge that PR?

Reply:
@jodh-intel @sameo Seems like I missed repushing a change I made yesterday:( 
Changes are in.